README.md

```diff
@@ -0,0 +1,62 @@
+
+# Upload your sections and articles to Zendesk  
+
+From a FileTree with folders and HTML articles, upload them to your Zendesk Help Center.  
+ 
+## **Goal**
+  
+The goal of this plugin is to provide a way to upload a whole FileTree containing sections & articles to Zendesk.  
```

Comment:
The goal of this plugin is to provide a way to upload a whole file tree containing sections & articles to Zendesk.